eng/pipelines/stress-http.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+pool:
+  name: Hosted VS2017
+
+variables:
+  _buildConfiguration: Release
+  httpStressProject: 'src/System.Net.Http/tests/StressTests/HttpStress/'
+  httpStressArgs: '-maxExecutionTime 20 -displayInterval 60'
+
+steps:
+- checkout: self
+  clean: true
+  fetchDepth: 1
+  lfs: false
+
+- task: CmdLine@2
+  displayName: Build Corefx
+  inputs:
+    script: |
+      .\build.cmd -ci -configuration $(_buildConfiguration)
+
+- task: PowerShell@2
+  displayName: Run HttpStress
+  inputs:
+    targetType: 'inline'
+    script: |
+      cd '$(Build.SourcesDirectory)/$(HttpStressProject)'
+      # Load testhost sdk in environment
+      . .\load-corefx-testhost.ps1 -c $(_buildConfiguration)
+      # Run the stress suite
+      dotnet run -c $(_buildConfiguration) -- $(httpStressArgs)
+
+- task: PublishBuildArtifacts@1
+  displayName: Publish Logs
+  inputs:
+    PathtoPublish: '$(Build.SourcesDirectory)/artifacts/log/$(_buildConfiguration)'
+    PublishLocation: Container
+    ArtifactName: 'httpstress_$(Agent.Os)_$(Agent.JobName)'
+  continueOnError: true
+  condition: always()
```

Comment:
Nit: extra add an extra line at the end.

Reply:
Is that convention? My VsCode trims that line by default (I also tend to prefer it like this).

Reply:
> Is that convention?

Yeah... it's why GitHub is showing that 
![image](https://user-images.githubusercontent.com/2642209/67031258-1f005700-f0df-11e9-975e-b11f86de1454.png)
Some text editors aren't happy when there's no ending newline.

Not a big deal, though.